# ObjectNode.msg

int32 id
float32[] points  # xyzrgb 포인트들
float32[] center
float32[] min_pt
float32[] max_pt
string class_name
int32 class_id
float32 conf
